### PR TITLE
Feat: Input suggestions

### PR DESCRIPTION
**Pull request by paulobernardoaf**
**Proposed changes**:
- Added input suggestions support

**Status (please check what you already did)**:
- [X] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: src/components/Widget/components/Conversation/components/Sender/components/Suggestions/styles.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    min-width: inherit;
+    background-color: grey;
+    padding-bottom: 2px;
+    padding-top: 2px;
+    min-height: 1.5em;
+  }
+
+  .push-suggestions-container {
+    background-color: var(--chatBackgroundColor);
+    border-top: 1px solid #b5b5b5;
```
Comment (ricardovasconcelos) [inline src/components/Widget/components/Conversation/components/Sender/components/Suggestions/styles.scss:20]: Top! único feedback é sempre usar as cores que estão no scss, nesse caso será o $grey-3

**Review by ricardovasconcelos — APPROVED** [review]

File: src/index.js [hunk 2]
```diff
@@ -240,6 +246,9 @@ ConnectedWidget.defaultProps = {
   showHeaderAvatar: true,
   sessionId: null,
   startFullScreen: false,
+  suggestionsUrl: 'https://api-staging.bothub.it/v2/repository/examples/search_repositories_examples/',
```
Comment (johncordeiro) [inline src/index.js:249]: Can you change the default value for URL for https://api.bothub.it instead of https://api-staging.bothub.it

**Review by johncordeiro — CHANGES_REQUESTED** [review]

**Review by johncordeiro — APPROVED** [review]

File: src/components/Widget/components/Conversation/components/Sender/components/Suggestions/index.js [hunk 3]
```diff
@@ -9,37 +9,54 @@ import { connect } from 'react-redux';
...
+    const element = this.suggestionsListRef.current;
+    // the + 10 is to add the margin from the prev and next buttons since offsetWidth doesn't include that
+    element.scrollLeft -= this.state.scrollDistance + 10;
+    this.setState({ scrollDistance: element.offsetWidth + 10 })
   };
   right = () => {
-    this.suggestionsListRef.current.scrollLeft += this.scrollDistance;
+    const element = this.suggestionsListRef.current;
+    // the + 10 is to add the margin from the prev and next buttons since offsetWidth doesn't include that
+    element.scrollLeft += this.state.scrollDistance + 10;
```
Comment (ricardovasconcelos) [inline src/components/Widget/components/Conversation/components/Sender/components/Suggestions/index.js:45]: The functions left and right appear to be the same, on function right() should be element.scrollRight ?
maybe it's better create just one function and get element.scroll by params

File: src/components/Widget/components/Conversation/components/Sender/components/Suggestions/index.js [hunk 4]
```diff
@@ -29,18 +29,13 @@ class SuggestionsList extends PureComponent {
     this.setState({ scrollDistance: element.offsetWidth + 10 })
   }
 
-  left = () => {
+  // direction is +1/-1 and represents right or left
+  scroll = (direction) => {
```
Comment (ricardovasconcelos) [inline src/components/Widget/components/Conversation/components/Sender/components/Suggestions/index.js:33]: Great!!

**Pull request merged**